openapi/business/business.yaml

```diff
@@ -0,0 +1,321 @@
+openapi: 3.0.0
+info:
+  title: Business REST APIs
+  version: '1.0'
+  description: APIs for managing data for a single business location.
+servers:
+  - url: 'https://prod.apigateway.co/business'
+    description: Production
+  - url: 'https://demo.apigateway.co/business'
+    description: Demo
+  - url: '{local}/products/reviews'
+    description: Local
+  - url: 'http://localhost:11001/business'
+    description: Localhost
+paths:
+  /customers:
+    post:
```

Comment:
You will need to define the `option` method as well.

Reply:
Yep, it's there, just further down.